reactive/zookeeper.py

```diff
@@ -0,0 +1,51 @@
+'''
+Reactive handlers for Zookeeper.
+'''
+
+from charmhelpers.core import hookenv
+from charms.layer.zookeeper import Zookeeper
+from charms.reactive import set_state, when, when_not
+
+@when('bigtop.available')
+@when_not('zookeeper.installed')
+def install_zookeeper():
+    '''
+    After Bigtop has done the initial setup, trigger a puppet install,
+    via our Zooekeeper library.
+
+    puppet will start the service, as a side effect.
+
+    '''
+    zookeeper = Zookeeper()
```

Comment:
At the very beginning of a reactive method we would want to inform the use/admin what is going on. So you could have a call similar to this: "hookenv.status_set('waiting', 'Installing Zookeeper')"